website-1.x/data/users.js

```diff
@@ -140,6 +140,13 @@ module.exports = [
     fbOpenSource: true,
     pinned: true,
   },
+  {
```

Comment:
The published version of Flux is not on Docusaurus (yet).

Reply:
Thanks, I removed it.